packages/bootstrap-vue-next/src/components/BTable/BTable.vue

```diff
@@ -556,75 +357,126 @@ const callItemsProvider = async () => {
   }
 }
 
-const toggleRowDetails = (tr: TableItem) => {
-  tr._showDetails = !tr._showDetails
-}
-
 const getFieldColumnClasses = (field: TableFieldObject) => [
-  field.class,
-  field.thClass,
   {
-    [`table-${field.variant}`]: field.variant !== null,
     'b-table-sortable-column': isSortable.value && field.sortable,
-    'b-table-sticky-column': field.stickyColumn,
   },
 ]
 
-const getFieldRowClasses = (field: TableFieldObject, tr: TableItem) => [
-  field.class,
-  field.tdClass,
-  tr?._cellVariants && tr?._cellVariants[field.key]
-    ? `table-${tr?._cellVariants[field.key]}`
-    : undefined,
-  {
-    [`table-${field.variant}`]: field.variant !== null,
-    'b-table-sticky-column': field.stickyColumn,
-  },
-]
+const getRowClasses = (
+  item: TableItem | null,
+  type: string
+): string | Array<any> | null | undefined => {
```

Comment:
```suggestion
): string | any[] | null | undefined => {
```
Use [] syntax over Array syntax